scio-avro/src/main/scala/com/spotify/scio/avro/syntax/ScioContextSyntax.scala

```diff
@@ -31,7 +31,7 @@ import scala.reflect.ClassTag
 import scala.reflect.runtime.universe._
 
 /** Enhanced version of [[ScioContext]] with Avro methods. */
-final class AvroScioContext(private val self: ScioContext) extends AnyVal {
+final class ScioContextOps(private val self: ScioContext) extends AnyVal {
```

Comment:
@regadas why rename? `AvroScioContext` seems to be appropriately named as in others like `BigQueryScioContext`

Reply:
these are `Ops` that are added to `ScioContext` and the package is `com.spotify.scio.avro.syntax`. Avro prefix is a bit redundant.

Reply:
#1763 is another example

Reply:
in regards to `BigQueryScioContext` will be proposing the same thing.